actors/verifreg/src/emit.rs

```diff
@@ -63,11 +64,13 @@ pub fn claim_removed(rt: &impl Runtime, id: ClaimID, claim: &Claim) -> Result<()
 pub fn build_verifier_balance_event(
     verifier: ActorID,
     balance: &DataCap,
+    client: Option<ActorID>,
 ) -> Result<ActorEvent, ActorError> {
     EventBuilder::new()
         .typ("verifier-balance")
         .field_indexed("verifier", &verifier)
         .field("balance", &BigIntSer(balance))
+        .field_indexed("client", &client)
```

Comment:
Given the list-of-pairs logical and physical representation of an event, we have two ways to represent an absent field:
- include a k/v pair but the value is nil
- don't include the key at all

I mostly would go for omitting the key rather than including it with a nil value. One argument against this is that it would make it harder to naively splat event k/v pairs into a static structure, e.g. with the handy schema deserialisation tools in some programming languages. However that kind of static schema doesn't work generally anyway - e.g. sector activation events have repeated keys for pieces in a sector. Some custom splatting library could of course be written to interpret repeated keys into lists, etc, but such software could also handle missing keys for options.

So.... omit the field instead?